### PR TITLE
2.4.3-RELEASE

### PR DESCRIPTION
**Pull request by twstuart**
Just need to update the POM when merge done

### CODE REVIEW

File: root/api/src/main/java/org/patientview/api/service/NhsChoicesService.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 /**
  * NHS Choices service, for retrieving data from NHS Choices
- *
+ * <p>
```
Comment (twstuart) [inline root/api/src/main/java/org/patientview/api/service/NhsChoicesService.java]: Checkstyle

File: root/api/src/main/java/org/patientview/api/client/nhschoices/NhsChoicesApiClient.java [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+                LOG.error("InterruptedException in NhsChoicesApiClient ", ie);
+            }
+        }
+
+        return allConditions;
+    }
+
+    /**
+     * Get GP details from NHS Choices API, used to update GpMaster url if url is not set.
+     * <p>
```
Comment (twstuart) [inline root/api/src/main/java/org/patientview/api/client/nhschoices/NhsChoicesApiClient.java]: Checkstyle <p>

File: root/api/src/main/java/org/patientview/api/client/nhschoices/NhsChoicesApiClient.java [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+    }
+
+    /**
+     * Helper method to build full url baseUrl + endpoint uri
+     */
+    private String buildFullUrl(String uri) {
+        return apiUrl + uri;
+    }
+
+    public static final class Builder {
```
Comment (twstuart) [inline root/api/src/main/java/org/patientview/api/client/nhschoices/NhsChoicesApiClient.java]: Move public methods above privates

Comment (p-maks) [reply root/api/src/main/java/org/patientview/api/client/nhschoices/NhsChoicesApiClient.java]: it's inner class, usually at very bottom of the main class

**Pull request merged**